### PR TITLE
SAMPLE: Student work on cinema history.

### PR DESCRIPTION
**Pull request by AVCharpentier**
We, Marina PERALTA FONSECA, Astrid ROYER DE VERICOURT, Sun SUN ,  Agathe ALLIROL--BINET, Antoine-Valentin CHARPENTIER, hereby grant to Hyperglosae maintainers the right to publish our contribution under the terms of any licenses the Free Software Foundation classifies as Free Software Licenses.

### CODE REVIEW

**Comment by benel** [comment]
@Hypertopic/histoire-du-cinema Could you please fil in your full names in the grant? Thanks.

File: samples/cinema_video.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "_id": "4e1a31e14b032f2fa9e161ee9b009125",
+    "dc_title": "Video Buster Keaton (Demo)",
+    "dc_creator": "Buster Keaton",
+    "dc_issued": "2023-02-21",
+    "text": " [![](https://www.profession-spectacle.com/wp-content/uploads/2016/01/Buster-Keaton.jpg)](https://www.youtube.com/watch?v=JRXkAhMYKEc)",
+    "dc_license": "https://creativecommons.org/licenses/by-sa/4.0/"
```
Comment (benel) [inline samples/cinema_video.json:7]: I doubt Buster Keaton chose a license created in 2001... ;)
It is either in the public domain (`CC-0`) either with no license ("All rights reserved" should be displayed).
If you are not sure of the legal status of the movie, choose the latter.

**Review by benel — CHANGES_REQUESTED** [review]
Thank you very much for your samples.
They are great examples for Hyperglosae.

There is however a little problem to fix concerning a license.

File: samples/cinema_video.json (hunk 1 above)
Comment (AVCharpentier) [reply samples/cinema_video.json]: It's done

File: samples/cinema_video.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "_id": "4e1a31e14b032f2fa9e161ee9b009125",
+    "dc_title": "Video Buster Keaton",
+    "dc_creator": "Buster Keaton",
+    "dc_issued": "2023-02-21",
+    "text": " [![](https://www.profession-spectacle.com/wp-content/uploads/2016/01/Buster-Keaton.jpg)](https://www.youtube.com/watch?v=JRXkAhMYKEc)",
+    "dc_license": "All rights reserved"
```
Comment (benel) [inline samples/cinema_video.json:7]: You don't have to put "All rights reserved" as a license. This is the default display when no license is set.

**Review by benel — CHANGES_REQUESTED** [review]
Thank you very much for your contribution.

Please remove "All rights reserved" since this is the default display when no license is set.

**Comment by AVCharpentier** [comment]
It should be good. At the same time, I noticed that the title of this document was not appropriate. I put the name of the movie "Sherlock Junior" instead of a generic name .

**Review by benel — APPROVED** [review]
Thank you very much for your contribution @Hypertopic/histoire-du-cinema. Your samples are now in the main branch.

**Pull request merged**